packages/web-runtime/src/App.vue

```diff
@@ -215,8 +215,12 @@ export default {
     }
   },
   watch: {
-    $route() {
-      this.appNavigationVisible = false
+    $route: {
+      immediate: true,
+      handler: function(to) {
+        document.title = this.extractPageTitleFromRoute(to)
+        this.appNavigationVisible = false
```

Comment:
Please delete this line. ;-) Doesn't belong in here. If the user is on a screen which is wide enough for having the left sidebar and the files list side by side, we don't want to force him into closing it.